pkg/registry/apis/provisioning/routes.go

```diff
@@ -118,6 +119,7 @@ func (b *APIBuilder) GetAPIRoutes() *builder.APIRoutes {
 	}
 }
 
+// TODO: why didn't we create a connector as we did before or have a separate file?
```

Comment:
connectors live under a resource -- this is at the top level

we *could* use a connector model -- but it is also kinda weird and requires a URL rewrite:
https://github.com/grafana/grafana/blob/main/pkg/services/apiserver/builder/helper.go#L48